### DISCUSSION TITLE
Link to a specific log line

### DISCUSSION THREAD

**Discussion by scottillogical** [grafana/grafana#40480] [Dashboards, Panels & Visualizations]
When using loki as a data source, I would like to be able to generate a link to a specific log line to share it with someone else.

**Comment by scottillogical** [comment]
dup of https://github.com/grafana/grafana/discussions/37621

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
